Jenkinsfile

```diff
@@ -72,19 +74,24 @@ def RunIntegrationTest(k, v) {
         script{
             try{
                 withEnv(["KUBE_VERSION=${v}"]){
-                    unstash 'repo'
+                    unstash 'repo-amd64'
                     echo "running tests on k8s version ${v}"
+                    sh 'sleep 60' //intermittent issue when running apt-get right after vm is up
                     sh 'tests/scripts/makeTestImages.sh load amd64'
-                    sh 'build/run make -j\$(nproc) build.common'
-                    sh "tests/scripts/kubeadm-dind.sh up"
-                    sh 'build/run make -j\$(nproc) test-integration SUITE=Smoke'
+                    sh "tests/scripts/kubeadm.sh up"
+                    try{
+                        sh '''#!/bin/bash
+                        set -o pipefail
+                        export KUBECONFIG=$HOME/admin.conf
+                        _output/tests/linux_amd64/smoke -test.v -test.timeout 1200s 2>&1 | tee _output/tests/integrationTests.log'''
+                    }
+                    finally{
+                        sh "mv _output/tests/integrationTests.log _output/tests/${k}_${v}_integrationTests.log"
+                    }
                 }
             }
             finally{
-                archive '_output/tests/*'
-                junit allowEmptyResults: true, keepLongStdio: true, testResults: '_output/tests/*.xml'
```

Comment:
it was trying to record it, but there are no results to record as there are no .xml files. below is a snippet from pipeline log

``` Pipeline] [gce_ci] step
[gce_ci] Recording test results
[gce_ci] None of the test reports contained any result ```